stats/lib/request-helpers.js

```diff
@@ -5,10 +5,10 @@ import pg from 'pg'
 
 /** @typedef {import('@filecoin-station/spark-stats-db').Queryable} Queryable */
 
-export const getDayAsISOString = (d) => d.toISOString().split('T')[0]
+export const getDateString = (d) => d.toLocaleDateString('en-CA')
 
-export const today = () => getDayAsISOString(new Date())
-export const yesterday = () => getDayAsISOString(new Date(Date.now() - 24 * 60 * 60 * 1000))
+export const today = () => getDateString(new Date())
+export const yesterday = () => getDateString(new Date(Date.now() - 24 * 60 * 60 * 1000))
```

Comment:
This may need a change depending on the outcome of https://github.com/filecoin-station/spark-stats/pull/210/files#r1742924839